windows/vhba_hot_add_remove/vhba_test_prepare.yml

```diff
@@ -22,3 +22,37 @@
   when:
     - test_disk_controller_type == "nvme"
     - test_purpose == "hot-add"
+
+# For hot-add sata test, set the boot disk label firstly since boot order may be changed after hot-add disk.
+- name: "Set boot disk label"
+  block:
+    - name: "Get VM VirtualDisk info"
+      include_tasks: ../../common/vm_get_device_with_type.yml
+      vars:
+        device_vim_type: vim.vm.device.VirtualDisk
+
+    - name: "Set the disk label list"
+      ansible.builtin.set_fact:
+        boot_disk_label_list: "{{ boot_disk_label_list|default([]) + [item.deviceInfo.label] }}"
```

Comment:
There should be spaces before and after `|`.

Reply:
Fixed. Thanks.